transformations.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
+import re
+
+
+class Field_Transformations:
+    def __init__(self, dataframe):
+        self.dataframe = dataframe
+        self.possition_abbr()
+        self.add_dept_codes()
+        self.parse_pay_code()
+        self.parse_location_code()
+        self.create_jobcoastcode()
+
+    def possition_abbr(self):
+        new_column = []
+        for result in self.dataframe["paycom_job_title"]:
+            if "(" in result:
+                value = result.split("/")[2]
+                new_column.append(value[:-1])
+            else:
+                new_column.append(None)
+        self.dataframe["position"] = new_column
+
+    def add_dept_codes(self):
+        dept_codes = pd.read_csv("dept_codes.csv")
```

Comment:
I think this is the element that we should consider pulling out of Python and putting into SQL as an LK_table. This would allow more analysts to be able to make updates to dept_codes if we have to change these over time. Let's talk thru tho because I could be missing something important here and perhaps there's a good case to be made to keep this on the Extract Load side.

Reply:
Agreed that it'd be much easier to add and edit department codes if they lived in a table in our DW. I can transfer this over. Would I just have to create a table out of the CSV by importing it?

Reply:
Agree that loading this into the data warehouse as a table and then joining on it within a view might be the simplest approach.